wlog/svclog/svc1log/svc1logtests/tests.go

```diff
@@ -362,6 +417,9 @@ func jsonOutputTests(t *testing.T, loggerProvider func(w io.Writer, level wlog.L
 			logEntry := buf.Bytes()
 			err := safejson.Unmarshal(logEntry, &gotServiceLog)
 			require.NoError(t, err, "Case %d: %s\nService log line is not a valid map: %v", i, tc.Name, string(logEntry))
+			logEntryRewrite, err := safejson.Marshal(gotServiceLog)
```

Comment:
This set of assertions parses the printed service log and rewrites it. If the bytes are not the same length then duplicate fields must have been printed.